test/federated/test_train_config.py

```diff
@@ -299,7 +299,7 @@ def test_send_model_and_loss_fn(workers):
     # TODO check reason for error (RuntimeError: This event loop is already running) when starting websocket server from pytest-asyncio environment
     # dataset = sy.BaseDataset(data, target)
 
-    # process_remote_worker = start_proc(WebsocketServerWorker, dataset=(dataset, dataset_key), **kwargs)
+    # process_remote_worker = start_remote_worker(id="async_fit", hook=hook, dataset=(dataset, dataset_key))
```

Comment:
We can leave the comment, but the syntax is no longer correct. start_remote_worker returns a tuple of values.

Reply:
indeed, thank you!